src/components/BookItem.jsx

```diff
@@ -13,13 +14,172 @@ const StyledBookItemContainer = styled.li`
+      line-height: normal;
+      letter-spacing: normal;
+      margin: 20px 0 0;
+      button {
+        background-color: transparent;
+        border: none;
+        outline: none;
+        color: #0290ff;
+        font-size: 0.875rem;
+        font-weight: 300;
+        font-stretch: normal;
+        font-style: normal;
+        line-height: normal;
+        letter-spacing: normal;
+        &:hover {
+          cursor: pointer;
+          text-decoration: underline;
+        }
+      }
+    }
+  }
+  .completed {
+    width: 20%;
+    display: flex;
+    flex-direction: row;
+    span {
+      font-size: 1.5rem;
+      font-weight: bold;
+    }
+  }
+  .vertical-seperator {
+    width: 20%;
+    height: 100%;
+  }
+  .chapter {
+    width: 7.875rem;
+    height: 1.125rem;
+    opacity: 0.5;
+    font-family: RobotoSlab;
+    font-size: 0.813rem;
+    font-weight: 300;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;
+    color: #121212;
+  }
+  .one {
+    width: 4.688rem;
+  height: 1.313rem;
+  font-family: RobotoSlab;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+  margin-bottom: 10px !important;
+  }
+  .del-button {
+    button {
+      border-radius: 5px;
+      background-color: #0290ff;
+      color: #fff;
+      outline: none;
+      border: none;
+      padding: 10px 20px;
+      &:hover {
+        cursor: pointer;
+        background-color: #0272c4;
+        color: #fff;
+      }
+    }
+  }
+  `;
 
 const BookItem = ({ itemProp, delBook }) => (
   <StyledBookItemContainer>
-    <div>{itemProp.title}</div>
-    <div>{itemProp.author}</div>
-    <button type="button" onClick={() => delBook(itemProp.id)}>Delete</button>
+    <div className="title-container">
+      <div className="category">{itemProp.category}</div>
+      <div className="title">{itemProp.title}</div>
+      <div className="author">{itemProp.author}</div>
+      <div className="actions">
+        <span>Comments</span>
+        <span>  |  </span>
+        <button type="button" onClick={() => delBook(itemProp.id)}>
+          <span>Remove</span>
+        </button>
+        <span>  |  </span>
+        <span>Edit</span>
+      </div>
+    </div>
+    <div className="completed">
+      <div className="complete-loader" />
+      <div style={{ marginLeft: '10px' }}>
+        <span>0%</span>
+        <br />
+        Completed
+      </div>
+    </div>
+    <div className="vertical-seperator">
+      <hr style={{ width: '1px', height: '70%', color: '#e8e8e8' }} />
+    </div>
+    <div>
+      <div>
+        <span className="chapter">CURRENT CHAPTER</span>
+        <br />
+        <span className="one">Chapter 1</span>
+      </div>
+      <div className="del-button">
+        <button type="button" onClick={() => delBook(itemProp.id)}>UPDATE PROGRESS</button>
+      </div>
+    </div>
```

Comment:
- [ ] Great job so far. However, little thing to work on before your project could be approved. This milestone is strickly for styling and your design must look exactly like that of zeplin design. Kindly create some margin between the marked areas on the screenshot below in order to match the Zeplin design.

### Your Design
![your-zeplin](https://github.com/Abdulwhbhussain/Bookstore-React-Microverse/assets/26597432/8e807d1d-87dc-43ce-8e0e-bae99d882f93)

### Zeplin Design
![zeplin](https://github.com/Abdulwhbhussain/Bookstore-React-Microverse/assets/26597432/ddc90b2f-9551-4495-ac1d-195c5188898d)